### ISSUE TITLE
Turn of output to stdout?

### ISSUE THREAD

**Issue by njohansson** [rackerlabs/recap#56]
I just received our first Rackspace server having recap installed so I have no prior experience with it.
I noticed that by default I receive a nightly e-mail at 01:00 with the output from recaplog which I've pinpointed to the last line in /etc/cron.d/recap:

```
# Pack and clear log files
0 1 * * * root /usr/sbin/recaplog
```

This e-mail provides no value to me so I'd rather like to shut it off. I noticed someone touching the same subject in #35 but it was closed?

What it comes down to is that you pass logging to tee in recaplog (https://github.com/rackerlabs/recap/blob/master/src/recaplog#L62) which outputs to stdout as well as the log file.

I'm aware I could just > /dev/null in the cron file but it could just as well be that a future upgrade then overwrites this again so maybe it'd be better to be able to pass a configuration directive to not print log messages to stdout?
Or maybe you have another idea of how to go about this?

Many thanks!

**Comment by carlwgeorge** [comment]
Thanks for the feedback @njohansson.  I was the one that closed #35, but for some reason I never saw that follow up comment after I closed it.  If this is the exact same issue, let's merge the issues (reopen #35 and close this one).

I'm not sure yet what the best way to approach this yet is, but I can tell you that adding `> /dev/null` to /etc/cron.d/recap will not be overwritten by future updates, because that file is marked as `config(noreplace)` in the spec file.

**Comment by njohansson** [comment]
That sounds perfectly fine - many thanks!
